README.md

```diff
@@ -592,6 +591,64 @@ config.optimization
   .usedExports(usedExports)
 ```
 
+#### Config optimization minimizers
+
+```js
+// Backed at config.optimization.minimizers
+config.optimization
+  .minimizer(name) : ChainedMap
+```
+
+#### Config optimization minimizers: adding
+
+_NOTE: Do not use `new` to create the minimizer plugin, as this will be done for you._
+
+```js
+config.optimization
+  .minimizer(name)
+  .use(WebpackPlugin, args)
+
+// Examples
+
+config.optimization
+  .minimizer('css')
+  .use(OptimizeCSSAssetsPlugin, [{ cssProcessorOptions: { safe: true } }])
+
+// Minimizer plugins can also be specified by their path, allowing the expensive require()s to be
+// skipped in cases where the plugin or webpack configuration won't end up being used.
+config.optimization
+  .minimizer('css')
+  .use(require.resolve('optimize-css-assets-webpack-plugin'), [{ cssProcessorOptions: { safe: true } }])
+
+```
+
+#### Config optimization minimizers: modify arguments
+
+```js
+config.optimization
+  .minimizer(name)
+  .tap(args => newArgs)
+
+// Example
+config
```

Comment:
Hi !
I think there is a small mistake here, as it should be `config.optimization` too :)

Reply:
@Gramatiik, ah you're right. Would you be up for opening a PR? :-)

Reply:
I've opened #159 :-)